src/main/java/co/getmehired/getmehired/rest/CommonDataResource.java

```diff
@@ -1,34 +1,36 @@
 package co.getmehired.getmehired.rest;
 
 import java.util.List;
-
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
-
 import co.getmehired.getmehired.model.Country;
 import co.getmehired.getmehired.model.Timezone;
 import co.getmehired.getmehired.model.University;
+import co.getmehired.getmehired.service.CommonDataService;
 
 @RestController
 public class CommonDataResource {
-	
+	@Autowired
+	private CommonDataService commondataservice;
+
 	@GetMapping("/api/timezones")
-	public List<Timezone> getTimezones() {
-		
-		return null;
+	public List<Timezone> getTimezones(@RequestParam String search) {
```

Comment:
For timezone we will send all as this will be used in dropdown. Remove the search keyword and make endpoint to return all timezones of the collection.